app/services/verify_subscriber_email_service.rb

```diff
@@ -3,8 +3,14 @@ class VerifySubscriberEmailService < ApplicationService
 
   class RatelimitExceededError < StandardError; end
 
-  MINUTELY_THRESHOLD = 3 # TODO: update this based on data and comment
-  HOURLY_THRESHOLD = 20 # TODO: update this based on data and comment
+  # This allows for up to 2 retries, to account for users
+  # not getting the email quickly enough and resubmitting.
+  MINUTELY_THRESHOLD = 4
+
+  # This allows for up to 5 retries per hour, to account
+  # for the user needing to return to manage their
+  # subscriptions on different devices in the same hour.
+  HOURLY_THRESHOLD = 6
```

Comment:
This feels a bit low, particularly since this is the less vulnerable end-point for spamming. Given an hour is quite a long time it feels realistic you could do a fair more than this. Does doubling it (or matching the other one at 11) seem reasonable?

Reply:
For the lack of specific data, yes. It now matches the other journey.